### PR TITLE
Add cleanup commands to kubectl Windows install

### PR DESCRIPTION
**Pull request by asarenkansah**
Expands on the fixes for https://github.com/kubernetes/website/issues/38702

Modifying the kubectl install docs for Windows per this issue: https://github.com/kubernetes/website/issues/38702.

Gives instructions about how to delete static files that will no longer be used after the installation is complete.

### CODE REVIEW

**Review by kundan2707 — COMMENTED** [review]
LGTM

File: content/en/docs/tasks/tools/install-kubectl-windows.md [hunk 1]
```diff
@@ -72,6 +72,12 @@ The following methods exist for installing kubectl on Windows:
    kubectl version --client --output=yaml
    ```
 
+1. After installing the plugin, clean up the installation files:
+
+   ```bash
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-windows.md:77]: This is PowerShell, not bash. Right?

File: content/en/docs/tasks/tools/install-kubectl-windows.md [hunk 2]
```diff
@@ -191,6 +197,12 @@ Below are the procedures to set up autocompletion for PowerShell.
 
    If you do not see an error, it means the plugin is successfully installed.
 
+1. After installing the plugin, clean up the installation files:
+
+   ```bash
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-windows.md:202]: This is also PowerShell, not bash. Right?

**Pull request merged**